src/seedu/addressbook/data/AddressBook.java

```diff
@@ -22,13 +24,14 @@
 
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
-
+    private final ArrayList<Tagging> allTaggings;
```

Comment:
great job putting the collection of taggings in AddressBook. Think about what advantage this would bring about.